compiler/bindings/python/iree/compiler/onnx.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2024 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import sys
+from .tools import onnx
+
+sys.modules[__name__] = onnx
```

Comment:
We probably don't want to use `onnx` as an unqualified name here or in the filenames - too much risk of conflicting with the `onnx` package itself. Could keep building on the `import_onnx` name or pick a new name like `compile_onnx` or `iree_onnx`.

Naming is hard :P